### PR TITLE
Fix labelPattern regex match to support css property label feature

### LINKED ISSUE #2311: What does the label do?
On https://emotion.sh/docs/labels there is an example of how emotion adds a human readable postfix to classnames. 
But `css-11z9jyx` is shown instead of something along `css-11z9jyx-some-name`. I'm not sure if it is a bug or a misunderstanding on my part when reading the documentation.

### PR DESCRIPTION
**Pull request by iChenLei**
**What**: When user set `autolabel` as 'never' in `.babelrc`(also `dev-only` in production mode), css property `label` not work, current official emotion-js doc show this bug. Hope this pr can fix #2311 

**Why**:  look this comment ->  https://github.com/emotion-js/emotion/issues/2311#issuecomment-814050482

**How**: just safe add semicolon `;` to `cssString` to supoort `labelPattern` match.

**Checklist**:

- [ ] Documentation
- [x] Tests
- [x] Code complete
- [x] Changeset 

I am not sure this `PR` is a suitable solution, feel free to close it by `emotion-js` maintainer. /cc @mitchellhamilton @Andarist

### CODE REVIEW

File: packages/serialize/src/index.js [hunk 1]
```diff
@@ -357,7 +357,10 @@ export const serializeStyles = function(
 
   let match
   // https://esbench.com/bench/5b809c2cf2949800a0f61fb5
-  while ((match = labelPattern.exec(styles)) !== null) {
+  // styles.replace(/;$/, '').concat(';') safe add semicolon to fix #2311
+  while (
+    (match = labelPattern.exec(styles.replace(/;$/, '').concat(';'))) !== null
```
Comment (Andarist) [inline packages/serialize/src/index.js:362]: Could you try an alternative fix?
```diff
-let labelPattern = /label:\s*([^\s;\n{]+)\s*;/g
+let labelPattern = /label:\s*([^\s;\n{]+)\s*(;|$)/g
```

Comment (iChenLei) [reply packages/serialize/src/index.js:362]: Of course, thanks for code review, when I try to solve this issue, I found many ways to fix, so I am confused which one is  correct and suitable solution.

Comment (iChenLei) [reply packages/serialize/src/index.js]: @Andarist Hi, I try the alternative fix, it works well. And I have a question, Is CI broken ?
netlify's documentaion website auto deploy looks like not ok ...
```bash
1:03:48 AM: 🎁 info building bundles!
1:03:57 AM: 🎁 success built bundles!
1:03:58 AM: > emotion-monorepo@10.0.0 build:site /opt/build/repo
1:03:58 AM: > cd site && yarn build:site
1:03:58 AM: yarn run v1.7.0
1:03:58 AM: $ gatsby build
1:04:02 AM: success open and validate gatsby-configs - 0.316s
1:04:03 AM: error
1:04:03 AM: Something went wrong installing the "sharp" module
1:04:03 AM: Cannot find module '../build/Release/sharp.node'
1:04:03 AM: Require stack:
1:04:03 AM: - /opt/build/repo/node_modules/sharp/lib/constructor.js
```
Maybe need to fix emotion-js infra CI/CD ?

Comment (Andarist) [reply packages/serialize/src/index.js]: Yes, something is off with Netlify's setup. I didn't have time to dig into this.

File: .changeset/famous-papayas-roll.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/serialize': patch
+---
+
+fix the labelPattern regex match
```
Comment (Andarist) [inline .changeset/famous-papayas-roll.md:5]:
```suggestion
Fixed appending manual labels to Babel-optimized style objects.
```

**Pull request merged**